### PR TITLE
Docs: Improve BSE methods section

### PR DESCRIPTION
**Pull request by MGraml**

### CODE REVIEW

File: docs/methods/properties/bandstructure_gw.md [hunk 1]
```diff
@@ -5,9 +5,8 @@ GW for molecules/condensed phase systems with CP2K. In DFT, the energy of a mole
 corresponds to an eigenvalue of the Kohn-Sham matrix. In GW, the procedure for getting the level
 energies is to first perform a DFT calculation (commonly with the PBE or PBE0 functional) to get the
 molecular orbital wavefunctions and then compute a new GW energy for the molecular orbitals of
-interest. For an introduction into the concept of GW, please have a look at the [GW Compendium] or
-read Sec. II and the introduction to Sec. III in
-[doi:10.1103/PhysRevB.87.235132](https://dx.doi.org/10.1103/PhysRevB.87.235132).
+interest. For an introduction into the concept of GW, please have a look at [Golze2019](#Golze2019)
```
Comment (oschuett) [inline docs/methods/properties/bandstructure_gw.md:8]: For citations you can actually just write `[](#Golze2019)`

File: docs/methods/properties/bandstructure_gw.md [hunk 2]
```diff
@@ -206,8 +205,8 @@ experimental ionization potential (12.62 eV). Benchmarks on molecules and solids
 self-consistency of eigenvalues in the Green's function G improves the agreement between the GW
 calculation and experiment. This scheme is called GW0@PBE and is our favorite GW flavor so far
 (experience based on nano-sized aromatic molecules). You can also check
-[this video](https://www.youtube.com/watch?v=1vUuethWhbs&t=5563s) or the [GW Compendium] on which
-DFT starting functional and which self-consistency scheme could be good for your system.
+[this video](https://www.youtube.com/watch?v=1vUuethWhbs&t=5563s) or [Golze2019](#Golze2019) on
```
Comment (oschuett) [inline docs/methods/properties/bandstructure_gw.md:208]: You can embed videos like this:

````
```{youtube} 1vUuethWhbs
---
url_parameters: ?start=5563
align: center
privacy_mode:
---
```
````

**Review by oschuett — COMMENTED** [review]
Just some minor comments. If you prefer I can also merge it as is.

**Comment by MGraml** [comment]
Thank you for pointing that out! I will do that in the future. For now, it would be nice, if you could merge this PR as is.

**Pull request merged**